listenbrainz/webserver/views/test/test_index.py

```diff
@@ -118,9 +118,13 @@ def test_menu_logged_in(self, mock_user_get):
         self.assertIn('iliekcomputers', data)
         self.assertIn('Import', data)
         # item in user menu
+
         self.assertIn('Your Listens', data)
         mock_user_get.assert_called_with(user['user_login_id'])
 
+        self.assertIn('My Listens', data)
```

Comment:
We've changed "Your Listens" to "My listens", so the "Your Listens" are no longer needed.

Reply:
Okay